src/service/db/alerts/alert/new.rs

```diff
@@ -377,7 +377,7 @@ mod super_cluster {
 
     /// Sends event to the super cluster queue indicating that all alert have
     /// been deleted from the database.
-    pub async fn emit_delete_all_event() -> Result<(), infra::errors::Error> {
+    pub async fn _emit_delete_all_event() -> Result<(), infra::errors::Error> {
```

Comment:
@patrickcsullivan looks like this helper function isn't used anywhere. i marked it so to make clippy happy.